src/app/services/api.service.ts

```diff
@@ -53,7 +53,7 @@ export abstract class ApiService<T extends ResourceModel<T>> {
     } else {
       // The backend returned an unsuccessful response code.
       // The response body may contain clues as to what went wrong,
```

Comment:
delete comments

Reply:
those comments are OK, as they clarify why this part of the code is needed.